webrtc.html

```diff
@@ -2141,47 +2122,30 @@ <h2>Attributes</h2>
               "idlAttrType"><a>RTCSessionDescription</a></span>, readonly,
               nullable</dt>
               <dd>
-                <p>The <code><a data-link-for=
-                "RTCPeerConnection">currentRemoteDescription</a></code>
-                attribute represents the last remote
-                <code><a>RTCSessionDescription</a></code> that was successfully
+                <p>The <dfn id=
+                "dom-peerconnection-currentremotedesc"><code>currentRemoteDescription</code></dfn>
+                attribute MUST return <a>[[\CurrentRemoteDescription]]</a>.</p>
+                <p>It represents the last remote description that was successfully
                 negotiated the last time the <code>RTCPeerConnection</code>
                 transitioned into the stable state plus any remote candidates
                 that have been supplied via <code><a data-link-for=
                 "RTCPeerConnection">addIceCandidate()</a></code> since the
                 offer or answer was created.</p>
-                <p>The <dfn id=
-                "dom-peerconnection-currentremotedesc"><code>currentRemoteDescription</code></dfn>
-                attribute MUST return the value that algorithms in this
-                specification set it to, complete with any remote candidates
-                that have been supplied via <code><a data-link-for=
-                "RTCPeerConnection">addIceCandidate()</a></code> since the
-                offer or answer was created. Prior to being set, it returns
-                null.</p>
```

Comment:
I note that this is a "red only" change, but that's fine since I think it's only repeating what WedIDL and the other procedures has defined already.